### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by comradelemoncake**
Tested locally and it runs great:

image

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# The Bartholomew documentation website
+
+First, build Bartholomew based on the
+[contributing guide](./content/docs/contributing.md).
```
Comment (radu-matei) [inline docs/README.md:4]: Missing file.

File: docs/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "spin-docs",
```
Comment (radu-matei) [inline docs/package.json]: This entire file should point to Bartholomew resources rather than Spin.

**Comment by radu-matei** [comment]
This should also remove the rest of the content from the root of the repository, since it's now in the `docs/` directory.

File: docs/content/bart-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
+title = "Building a site with Bartholomew"
```
Comment (radu-matei) [inline docs/content/bart-tutorial.md]: What is this file?
It seems to be the Spin URL shortener tutorial, but with a new title?

File: docs/content/contributing.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
+title = "Contributing to Spin"
```
Comment (radu-matei) [inline docs/content/contributing.md]: This file should not be here.

Comment (comradelemoncake) [reply docs/content/contributing.md]: Are you sure? I think we should probably have a contributing guide for the docs. This PR was more for tracking before we have a greater conversation about the structure of the docs.

File: docs/content/bart-tutorial.md (hunk 3 above)
Comment (comradelemoncake) [reply docs/content/bart-tutorial.md]: As previously stated, this is the initial structure for the Bartholomew docs. This isn't the final product. I believe we should have a larger discussion about how we want things structured.

File: docs/content/index.md [hunk 5]
```diff
@@ -22,6 +22,4 @@ With all the buzz around cloud-native WebAssembly, the team at Fermyon was influ
 
 The result: An adaptable WebAssembly CMS that uses a Functions-as-a-Service model for page rendering instead of requiring the entire site to be pre-generated to static HTML.
 
-Bartholomew is built as a [Wagi](https://github.com/deislabs/wagi) application and will run inside of [Hippo](https://github.com/deislabs/hippo) or inside of [Krustlet](https://krustlet.dev/) if you have the Wagi extension. At Fermyon, we run our website using Hippo deploying to a [Nomad](https://www.nomadproject.io/) cluster.
```
Comment (radu-matei) [inline docs/content/index.md]: Bartholomew is no longer a Wagi application — see #65

**Comment by radu-matei** [comment]
I think you mistakenly deleted the `bart` directory as well.

**Review by radu-matei — CHANGES_REQUESTED** [review]
My earlier comment was around removing the duplicate content that is now also part of the `docs` directory, so before another round of reviews, could you please add everything else back?



(readme, license, GitHub workflows, source code)

**Comment by radu-matei** [comment]
Ok, I will try to clarify from the beginning.

The `main` branch of the Bartholomew repository contains `config`, `content`, `scripts`, `static`, and `templates` in the root.
Those directories are supposed to actually be rended in our docs website, so you initially copied them over in the `docs` directory. But by copying them, we now have duplicated content in the root of the repository and in `docs`, and it is unclear which is supposed to be the source of truth for our content, and which one should update.

So in my first comment, I said:

> This should also remove the rest of the content from the root of the repository, since it's now in the docs/ directory.

"Content" there meaning the duplicated content — I apologize, that comment should have been a bit more clear.

You then removed everything from the root of the repository, including the GitHub workflows, license, readme, and source code.

Now, you re-added *some* of them back (but not all — the GitHub workflows are missing, for example), and re-added `config`, `scripts`, `static`, and `templates` back to the root of the repository.
Also, the re-added source code (the `bart` directory) is modified — it is missing the new line at the end of each file (the fix here would be to reset the changes to those files, as opposed to modifying the files again).

There is a `.DS_Store` file that we should probably add to `.gitignore`. The file `bart/Cargo.toml` was renamed to `bart/cargo.toml`.

There is also some content that was not moved to the `docs` directory, perhaps on purpose, but wanted to mention it — mainly `content/atom.md`, and pages under `content/docs/`.

Hopefully this is useful, let me know if I can help further.

**Comment by radu-matei** [comment]
Added in #74

**Pull request closed without merging**